daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -958,10 +979,10 @@ private[lf] object SBuiltin {
     *    -> CachedContract
     *    -> ContractId arg
     */
-  final case object SBUCreate extends OnLedgerBuiltin(2) {
-    override protected def executeWithLedger(
+  final case object SBUCreate extends UpdateBuiltin(2) {
```

Comment:
Nit pick: we obviously have a naming convention here (e.g. `SBUXXX` vs `SBXXX` vs `SBSXXX` - would there be any value in documenting this desire within this file?

Reply:
I look a bit more precisely, and in fact there is actually no a proper convention. 
SBU concern more Update primitives, and SBS more Scenario primitives but that is actually it. 
It is in fact pretty difficult to come with a proper convention due the way we are compilation.
- SBUxxx is an implementation of an Ast.UpdateXXX primitive

Reply:
Ok, fair enough. Thanks for looking.

Reply:
Will do that in another PR.